fixed-hash/src/hash.rs

```diff
@@ -636,6 +637,42 @@ macro_rules! impl_quickcheck_for_fixed_hash {
 	};
 }
 
+// Implementation for disabled rand crate support.
+//
+// # Note
+//
+// Feature guarded macro definitions instead of feature guarded impl blocks
+// to work around the problems of introducing `arbitrary` crate feature in
+// a user crate.
+#[cfg(not(feature = "arbitrary"))]
+#[macro_export]
+#[doc(hidden)]
+macro_rules! impl_arbitrary_for_fixed_hash {
+	( $name:ident ) => {};
+}
+
+// Implementation for enabled rand crate support.
```

Comment:
```suggestion
// When the `arbitrary` features is disabled.
```

Reply:
Done :)

Reply:
you meant enabled :P
to be consistent with other macro impls, I would prefer to keep it, but fix the copy-paste typo `s/rand/arbitrary`